src/bundle/Resources/public/scss/_general.scss

```diff
@@ -39,7 +39,7 @@ body {
     height: auto;
 }
 
-button {
+button, a {
```

Comment:
Maybe we should remove the `button, a` part and make the `.btn` styles more generic? cc @dew326

Reply:
Good point, +1